### ISSUE TITLE
[Bug]: The flashcard screen doesn't display

### ISSUE THREAD

**Issue by art-kslv** [st3v3nmw/obsidian-spaced-repetition#1070]
### Describe the bug

I'm trying to open Spaced Repetition for the first time, and it doesn't open correctly. 
![image](https://github.com/user-attachments/assets/708869d1-9e8f-4e5b-bbf0-d1f6dddbfd40)
Open in bottom right corner. Buttons are not visible

### To Reproduce

Press the flashcard button

### Expected behavior

Opens correctly in the middle

### Screenshots

![image](https://github.com/user-attachments/assets/77c73535-2f81-47f6-beeb-cca7a4aa7d28)

### OS of your device

Win 11

### Obsidian version

v1.6.7

### Plugin version

v1.12.5

**Comment by ronzulu** [comment]
Hi there, this may be similar to [ [Bug]: Review card position error #1036 ](https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/1036)

In that case it only occurred for a very specific theme. What theme are you using? If non-standard, does the problem still occur if you change the theme to a standard one?

Cheers
Ronny

**Comment by art-kslv** [comment]
Yes, when I changed to the default color theme, the bag disappeared
